e2e/portalicious/tests/RegistrationPage/ViewRegistrationSummary.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { expect, test } from '@playwright/test';
+
+import { SeedScript } from '@121-service/src/scripts/enum/seed-script.enum';
+import {
+  searchRegistrationByReferenceId,
+  seedPaidRegistrations,
+} from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+} from '@121-service/test/helpers/utility.helper';
+import { registrationPvScoped } from '@121-service/test/registrations/pagination/pagination-data';
+
+import LoginPage from '@121-e2e/portalicious/pages/LoginPage';
+import RegistrationActivityLogPage from '@121-e2e/portalicious/pages/RegistrationActivityLogPage';
+
+const projectId = 2;
+let registrationId: number;
+let registrationProgramId: number;
+
+test.beforeAll(async ({}) => {
+  await resetDB(SeedScript.nlrcMultiple);
+
+  const accessToken = await getAccessToken();
+  await seedPaidRegistrations([registrationPvScoped], projectId);
+  const searchRegistrationResponse = await searchRegistrationByReferenceId(
+    registrationPvScoped.referenceId,
+    projectId,
+    accessToken,
+  );
+  const registration = searchRegistrationResponse.body.data[0];
+  registrationId = registration.id;
+  registrationProgramId = registration.registrationProgramId;
+});
+
+test('User should see a summary of a registration', async ({ page }) => {
+  const loginPage = new LoginPage(page);
+  await page.goto(`/`);
+  await loginPage.login(
+    process.env.USERCONFIG_121_SERVICE_EMAIL_ADMIN,
+    process.env.USERCONFIG_121_SERVICE_PASSWORD_ADMIN,
+  );
+  const activityLogPage = new RegistrationActivityLogPage(page);
+  await activityLogPage.goto(
+    `/project/${projectId}/registrations/${registrationId}`,
```

Comment:
This redirects to `/activity-log` do we want to do an assert on that?

I tested the UI, which links to `/project/${projectId}/registrations/${registrationId}` so let's keep on doing that in the test as well.